VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-list.component.ts

```diff
@@ -64,9 +73,9 @@ export class JudgeHearingListComponent implements OnInit, OnDestroy {
     this.conferencesSubscription = this.videoWebService.getConferencesForJudge().subscribe((data: ConferenceForUserResponse[]) => {
       this.loadingData = false;
       this.conferences = data;
-        if (this.conferences.length > 0) {
```

Comment:
more white space commits

Reply:
We will pick up standardizing white space as a task. Will put that into the next sprint